serving/docker/dockerd-entrypoint-with-cuda-compat.sh

```diff
@@ -32,6 +32,14 @@ else
     echo "Skip CUDA compat libs setup as package not found"
 fi
 
+if [ -n "$SM_NEO_EXECUTION_CONTEXT" ]; then
```

Comment:
So customer only provide an env and by default we enter this path to compile with Neo? Do we want to check what is inside this env vars?

Reply:
In practice, customers won't set this env var themselves. It will be set by the Neo compilation host when running the container. We could explicitly require it to be set to "1" if you think that would be preferable